packages/frontend/package.json

```
@@ -13,11 +13,12 @@
     "@aws-sdk/s3-request-presigner": "3.627.0",
     "@aws-sdk/util-create-request": "3.622.0",
     "@aws-sdk/util-format-url": "3.609.0",
-    "@reach/router": "1.3.4",
+    "@reach/router": "^1.3.4",
```

I don't think `@reach/router` is required to be updated to bump React version.
```suggestion
    "@reach/router": "1.3.4",
```

It's latest version is `1.3.4` https://www.npmjs.com/package/@reach/router?activeTab=versions, as it's deprecated in favor of React Router.
That migration is tracked in https://github.com/aws-samples/aws-sdk-js-notes-app/issues/26